lib/audited.rb

```diff
@@ -13,7 +13,11 @@ class << self
     attr_writer :audit_class
 
     def audit_class
-      @audit_class ||= Audit
+      @audit_class ||= "Audited::Audit"
```

Comment:
I think this is actually a breaking change and should not have been released as a minor version. It's really odd to have a method named `class` return a string.